content/200-concepts/100-components/01-prisma-schema/06-relations/200-one-to-many-relations.mdx

```diff
@@ -45,12 +51,55 @@ model Post {
 
 > **Note** The `posts` field does not "manifest" in the underlying database schema. On the other side of the relation, the [annotated relation field](/concepts/components/prisma-schema/relations#annotated-relation-fields-and-relation-scalar-fields) `author` and its relation scalar `authorId` represent the side of the relation that stores the foreign key in the underlying database.
 
-This 1-n-relation expresses the following:
+This one-to-many relation expresses the following:
 
 - "a user can have zero or more posts"
 - "a post must always have an author"
 
-</TopBlock>
+In this example, the `author` field of the `Post` model references the `id` field of the `User` model. It is also possible to reference another field, as long as that field is marked with the `@unique` attribute. This is necessary to guarantee that there is only a single `User` connected to each `Post`. In the following example, the `author` field references an `email` field in the `User` model, which is marked with the `@unique` attribute:
+
+<TabbedContent tabs={[<FileWithIcon text="Relational databases" icon="database"/>, <FileWithIcon text="MongoDB" icon="database"/>]}>
+<tab>
+
+```prisma
+model User {
+  id    Int    @id @default(autoincrement())
+  email String @unique // <-- add unique attribute
+  posts Post[]
+}
+
+model Post {
+  id       Int  @id @default(autoincrement())
+  authorId Int
+  author   User @relation(fields: [authorId], references: [email])
+}
+```
+
+</tab>
+<tab>
+
+```prisma
+model User {
+  id    Int    @id @default(auto()) @map("_id") @db.ObjectId
+  email String @unique // <-- add unique attribute
+  posts Post[]
+}
+
+model Post {
+  id       Int  @id @default(auto()) @map("_id") @db.ObjectId
+  authorId Int
+  author   User @relation(fields: [authorId], references: [email])
+}
+```
+
+</tab>
+</TabbedContent>
+
+<Admonition type="warning">
+
+In MySQL, it is possible to create a foreign key with only an index on the referenced side, and not a unique constraint. In Prisma versions 4.0.0 and later, if you introspect a relation like this it will trigger a validation error. To fix this, you will need to add a `@unique` constraint to the referenced field.
```

Comment:
```suggestion
In MySQL, it is possible to create a foreign key with only an index on the referenced side, and not a unique constraint. In Prisma versions 4.0.0 and later, if you introspect a relation like this it will trigger a validation error. To fix this, you will need to add a `@unique` constraint to the referenced field.
```

By "introspect a relation like this", do you mean "introspect a relation of this type"? Or do you mean "introspect a relation in this way"? Can this be clarified?